README.md

```diff
@@ -1,69 +1,176 @@
+composer install
+```
 
+With Docker, install dependencies with `composer` image:
 ```bash
 docker-compose up
-docker-compose run -u www-data --rm php php composer install
+docker run --rm --interactive --tty --volume $PWD:/app --volume ${HOST_COMPOSER_HOME:-~/.composer}:/var/www/.composer composer:1.7 install
 ```
 
-# Setup
-Note: to setup this tool, you'll need a valid **API Client ID** and its **API Client secret** from your Akeneo PIM instance. Read the dedicated documentation to proceed: https://api.akeneo.com/getting-started-admin.html
+You're ready to migrate your assets!
+
+## Setup
 
 You need to copy the [.env](https://symfony.com/doc/current/components/dotenv.html) file:
 ```bash
 cp .env .env.local
 ```
 
 Then open `.env.local` to define the needed configuration vars:
-```
-AKENEO_API_BASE_URI=http://your-akeneo-pim-instance.com
-```
+- `AKENEO_API_BASE` refers to the URL of your PIM Enterprise Edition, used for API calls.
+   For example, `http://localhost:80`.
+   If you use Docker, set this value to `http://httpd:80` (or `https://httpd:443` if you use SSL).
+- `APP_ENV` refers to the `APP_ENV` of your PIM Enterprise Edition, used for direct bask calls.
+   Set it to `prod`, `prod_onprem_paas`...
 
-## Docker setup [optional]
+## Docker setup
 
-- Update your Enterprise Edition docker-compose file to allow external data from network
+First, you have to update your Enterprise Edition `docker-compose.yml` file to allow external data from network.
 
 ```yaml
 networks:
   pim:
     external: true
 ```
+Then you will have to recreate this network manually. 
+Go to your Enterprise Edition directory.
+List the current networks with
+```bash
+docker network list
+```
+Find your Enterprise Edition network in this list, stop your Docker, then removes it, to recreate the new one.
+```bash
+docker-container stop
+docker network create pim
+docker-container up -d
+```
+
+Then, you have to use the `docker-compose.override.yml` of this repository. 
+Copy paste the `docker-compose.override.yml.dist` into `docker-compose.override.yml`.
+Then, update `/path/to/ee/` to match to your local installation (e.g. `/home/akeneo/pim-ee/`).
+Finally, restart your CsvToAsset docker:
+```bash
+docker-compose up
+```
+
+## How to Use
+
+Please read the different scenarios to find the one which will match your needs.
+
+### Full-automatic migration
+
+This mode is for a PIM with a simple usage of the former assets.
+It will automatically create your previous assets and import them it into one unique new Asset family.
+As the categories and tags does not exist in the new Asset feature, we will keep them in a specific field on each Asset.
+
+To migrate your data, run
+```bash
+./bin/migrate.php <family-code> <path-to-ee-installation>
+```
+with
+- `family-code` will be the code of your brand new Asset family,
+- `path-to-ee-installation` is the path of your local Enterprise Edition. If you use Docker, this value will be `/srv/ee`.
+
+### Migration semi-automatic
```

Comment:
```suggestion
### Migration family by family
```

Reply:
I'm not fan of "family by family". For me the "family by family" is the "full manual" mode